WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -193,7 +193,7 @@ class PlanDetailViewController: UIViewController {
     
     private func populateHeader() {
         let plan = viewModel.plan
-        planImageView.image = plan.image
+        planImageView.setImageWithURL(plan.iconUrl)
```

Comment:
I think we should be using the active icon URL here if the plan is active. Also, should we resize the icon here to ensure it's the correct size?